### PR TITLE
Bootstrap plugin

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we apply the `io.spine.tools.gradle.bootstrap` plugin (a.k.a. [Bootstrap](https://github.com/SpineEventEngine/bootstrap) plugin) to this project.

With the help of the plugin, we're able to hide some tedious configuration details from the user.

Also, in this PR we split the build script into smaller case-specific configuration parts. The helper scripts and resources required by those scripts are now located in the `script` subfolder of the project.

Also, in this PR we perform a small update on the repository README.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: server/build.gradle [hunk 1]
```diff
@@ -18,10 +18,11 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+spine.java().server()
```
Comment (armiol) [inline server/build.gradle]: Let's discuss the wording tomorrow.

File: build.gradle [hunk 2]
```diff
@@ -17,140 +17,56 @@
...
-        options.errorprone.errorproneArgs << "-XepExcludedPaths:.*/generated/.*" << "-Xep:ClassCanBeStatic:OFF"
-    }
+    apply from: "$rootDir/script/pmd/pmd.gradle"
+    apply from: "$rootDir/script/idea.gradle"
+    apply from: "$rootDir/script/tests.gradle"
 
     sourceCompatibility = 1.8
     targetCompatibility = 1.8
 
+    configurations {
```
Comment (armiol) [inline build.gradle:48]: This is something we would want to configure with the plugin. It's really a magical piece to those who never had a word with Spine. Also, Spine does not seem to work properly, if these aren't excluded.

**Review by armiol — COMMENTED** [review]
@dmdashenkov PSMC.

File: build.gradle (hunk 2 above)
Comment (dmdashenkov) [reply build.gradle:48]: Please see [this issue](https://github.com/SpineEventEngine/bootstrap/issues/6) in `bootstrap` for the further improvements.

We will eliminate the need to configure these things, in an iterative manner.

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

**Review by armiol — APPROVED** [review]

**Pull request merged**